Makefile

```diff
@@ -369,31 +369,41 @@ devdoc-update:
 ###                                Protobuf                                 ###
 ###############################################################################
 
+containerProtoVer=latest  # v0.1
```

Comment:
before merging we need to tag a new release for the container, because swagger-gen only works with latest.

Reply:
you should be able to modify the github action to push once with a tag, then you revert it prior to merging this.

Reply:
that was hack. Can we have a human maintainer for the docker hub?

Reply:
yea, many people have it. But why not just use the bot? It helps with avoiding the permission hell that arises